py-polars/polars/interchange/__init__.py

```diff
@@ -1,5 +1,5 @@
 """
-Module containing the implementation of the Python dataframe interchange protocol.
+Module containing the implementation of the Python DataFrame interchange protocol.
```

Comment:
Again this must remain lowercase:
https://data-apis.org/dataframe-protocol/latest/index.html

Reply:
Doh; thought I _had_ left all of those alone, for exactly the reason given. A few rogue replacements must have made it through - will fix 🤦

Reply:
Updated/reverted these👌